src/nodeFinders.ts

```diff
@@ -0,0 +1,53 @@
+import { SyntaxNode } from "web-tree-sitter";
+import { NodeFinder } from "./Types";
+
+export const findNode =
+  (isTargetNode: (node: SyntaxNode) => boolean): NodeFinder =>
+  (node: SyntaxNode) => {
+    return isTargetNode(node) ? node : null;
+  };
+
+export const findNodeOfType =
```

Comment:
`typedNodeFinder`?  Idk maybe verb convention is better than noun.  But this does return a finder, it doesn't find.  I don't feel strongly as long as we're consistent